### PR TITLE
brotli: update page

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

From #9578

### CODE REVIEW

File: pages/common/brotli.md [hunk 1]
```diff
@@ -5,20 +5,20 @@
 
 - Compress a file, creating a compressed version next to the file:
 
-`brotli {{file.ext}}`
+`brotli {{path/to/file.ext}}`
```
Comment (pixelcmtd) [inline pages/common/brotli.md:8]: How about this?

```suggestion
`brotli {{path/to/file}}`
```

File: pages/common/brotli.md [hunk 2]
```diff
@@ -5,20 +5,20 @@
...
 
 - Decompress a file, creating an uncompressed version next to the file:
 
-`brotli -d {{file.ext}}.br`
+`brotli -d {{path/to/file.ext.br}}`
```
Comment (pixelcmtd) [inline pages/common/brotli.md:12]:
```suggestion
`brotli -d {{path/to/file.br}}`
```

File: pages/common/brotli.md [hunk 3]
```diff
@@ -5,20 +5,20 @@
...
 
 - Compress a file specifying the output filename:
 
-`brotli {{file.ext}} -o {{compressed_file.ext.br}}`
+`brotli {{path/to/file.ext}} -o {{path/to/compressed_output_file.ext.br}}`
```
Comment (pixelcmtd) [inline pages/common/brotli.md:16]:
```suggestion
`brotli {{path/to/file}} -o {{path/to/compressed_output_file.br}}`
```

File: pages/common/brotli.md [hunk 4]
```diff
@@ -5,20 +5,20 @@
...
 
 - Decompress a Brotli file specifying the output filename:
 
-`brotli -d {{compressed_file.ext.br}} -o {{file.ext}}`
+`brotli -d {{path/to/compressed_file.ext.br}} -o {{path/to/output_file.ext}}`
```
Comment (pixelcmtd) [inline pages/common/brotli.md:20]:
```suggestion
`brotli -d {{path/to/compressed_file.br}} -o {{path/to/output_file}}`
```

File: pages/common/brotli.md [hunk 5]
```diff
@@ -5,20 +5,20 @@
...
 
-- Specify the compression level. 1=Fastest (Worst), 11=Slowest (Best):
+- Specify the compression level [1=Fastest (Worst), 11=Slowest (Best)]:
 
-`brotli -q {{11}} {{file.ext}} -o {{compressed_file.ext.br}}`
+`brotli -q {{11}} {{path/to/file.ext}} -o {{path/to/compressed_output_file.ext.br}}`
```
Comment (pixelcmtd) [inline pages/common/brotli.md:24]:
```suggestion
`brotli -q {{11}} {{path/to/file}} -o {{path/to/compressed_output_file.br}}`
```

**Review by blueskyson — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**